examples/Request.re

```diff
@@ -14,81 +14,19 @@ Logs.set_reporter(Logs_fmt.reporter());
 
 */
 
-let https_url = "https://api.github.com/users/ostera";
+let https_url = "https://api.npms.io/v2/package/bsdoc";
 Logs.app(m => m("Requesting: %s", https_url));
 switch (
-  Httpkit_lwt.Client.(
-    Httpkit.Client.Request.create(
-      ~headers=[("User-Agent", "Reason HttpKit")],
-      `GET,
-      https_url |> Uri.of_string,
-    )
-    |> Https.send
-    >>= Response.body
-    |> Lwt_main.run
+  Httpkit.Client.Request.create(
+    ~headers=[("User-Agent", "Reason HttpKit")],
+    `GET,
+    https_url |> Uri.of_string,
   )
+  |> Httpkit_lwt_unix_h2.Client.Https.send(~client=`No_authentication)
+  >>= Httpkit_lwt_unix_h2.Client.Response.body
+  |> Lwt_main.run
```

Comment:
@ulrikstrid very similar api as before for requests — but that module name is just _too much_ xD got any ideas?